test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -1534,14 +1534,19 @@ namespace iroha {
       CHECK_ERROR_CODE_AND_MESSAGE(cmd_result, 5, query_args);
     }
 
+    class SetSettingValueTest : public CommandExecutorTest {
```

Comment:
This declaration should be moved to tests that use it (or vice versa).

Reply:
fixed